### PR TITLE
[Add] Deployment Configurations for website 

### PR DESCRIPTION
**Pull request by verma-kunal**
- Adding a deployment workflow, which would automate the site deployment on gh-pages, when new changes are done
- Modifying the config file to match with the deployment to this repository

### CODE REVIEW

**Comment by saiyam1814** [comment]
is the PR complete and ready to merge @verma-kunal ?

**Comment by verma-kunal** [comment]
> is the PR complete and ready to merge @verma-kunal ?

Yes @saiyam1814 !! Tested this on my system & the website was working on gh-pages
Hoping it works here as well :)

**Comment by AvineshTripathi** [comment]
@verma-kunal why are we keeping `ignore` to broken links

**Comment by verma-kunal** [comment]
> @verma-kunal why are we keeping `ignore` to broken links inside config file

Yep! So the reason is as we have deleted the `/blog` and some files in `/tutorial` folder as well. This was actually causing many broken link errors while deploying the site! It was actually due to this property `onBrokenLinks: "throw"`.
So, for the time being, till we set the content of the site, setting the property as `ignore`, to avoid any hinderance while deploying!

Let me know your thoughts :)

File: docusaurus.config.js [hunk 1]
```diff
@@ -8,9 +8,9 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 const config = {
   title: "Kubesimplify",
   tagline: "Simplifying Cloud Native for All",
-  url: "https://github.com",
-  baseUrl: "/kubesimplify/",
-  onBrokenLinks: "throw",
+  url: "https://kubesimplify.github.io",
+  baseUrl: "/website/",
+  onBrokenLinks: "ignore",
```
Comment (AvineshTripathi) [inline docusaurus.config.js:13]:
```suggestion
  onBrokenLinks: "ignore", //  We will change this to "throw" when proper setup is done
```

Comment (AvineshTripathi) [reply docusaurus.config.js:13]: just to make sure we don't miss this

Comment (verma-kunal) [reply docusaurus.config.js:13]: Nice idea

**Comment by AvineshTripathi** [comment]
LGTM!

**Comment by verma-kunal** [comment]
I think this one's ready for a merge 
cc @saiyam1814

**Pull request merged**